contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -67,6 +67,55 @@ add_v17_message_authorizations() {
     ' $json_file > temp.json && mv temp.json $json_file
 }
 
+
+add_emissions_withdraw_authorizations() {
```

Comment:
Small note here: any alternative that could work in Go?

Ideally we want to reduce the shell codebase and have everything in Go

Reply:
Okay, we can just use the same logic in Go a function ,
or a command similar to `add-observer-list` 

But I guess it's okay for now since we are adding just one authorization for now. If we need to modify more items in the genesis file, we can consider baking that into Zetacored in the future